tests/api/items/delete_by_ids.test.coffee

```diff
@@ -0,0 +1,76 @@
+CONFIG = require 'config'
+__ = CONFIG.universalPath
+_ = __.require 'builders', 'utils'
+should = require 'should'
+{ authReq, authReqB, getUser, undesiredRes } = require '../utils/utils'
+{ CountChange } = require './helpers'
+{ createItem } = require '../fixtures/items'
+debounceDelay = CONFIG.itemsCountDebounceTime + 100
+
+deleteByIds = (ids, authReqFn)->
+  authReqFn or= authReq
+  ids = _.forceArray ids
+  authReqFn 'post', '/api/items?action=delete-by-ids', { ids }
+
+describe 'items:delete-by-ids', ->
+  it 'should delete an item', (done)->
+    createItem()
+    .then (item)->
+      { _id: itemId } = item
+      deleteByIds itemId
+      .then (res)->
+        res.ok.should.be.true()
+        authReq 'get', "/api/items?action=by-ids&ids=#{itemId}"
+        .then (res)->
+          res.items.length.should.equal 0
+          done()
+    .catch done
+
+    return
+
+  it 'should trigger an update of the users items counters', (done)->
+    createItem()
+    # Delay to let the time to the item counter to be updated
+    .delay debounceDelay
+    .then (item)->
+      getUser()
+      .then (userBefore)->
+        deleteByIds item._id
+        # Delay to request the user after its items count was updated
+        .delay debounceDelay
+        .then (res)->
+          getUser()
+          .then (userAfter)->
+            countChange = CountChange userBefore.snapshot, userAfter.snapshot
+            countChange('private').should.equal 0
```

Comment:
i would delete this line and the next one, as `private` & `network` snapshot will stay unchange no matter what since the only [tested Item](tests/api/fixtures/items.coffee#L37) is `public`